components/cluster/pcs/files/pcsd.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version='1.0'?>
+<!DOCTYPE service_bundle SYSTEM '/usr/share/lib/xml/dtd/service_bundle.dtd.1'>
+<service_bundle type='manifest' name='PCSD_WebGUI'>
+  <service name='application/pcsd' type='service' version='1'>
+      <create_default_instance enabled='false'/>
+      <single_instance/>
+      <dependency
+         name='multi-user-server'
+         grouping='optional_all'
+         type='service'
+         restart_on='none'>
+         <service_fmri value='svc:/milestone/multi-user-server' />
+      </dependency>
+      <dependency name='network' grouping='require_all' restart_on='error' type='service'>
+        <service_fmri value='svc:/milestone/network:default'/>
+      </dependency>
+      <dependency name='filesystem-local' grouping='require_all' restart_on='none' type='service'>
+        <service_fmri value='svc:/system/filesystem/local:default'/>
+      </dependency>
+      <dependency name='autofs' grouping='optional_all' restart_on='error' type='service'>
+        <service_fmri value='svc:/system/filesystem/autofs:default'/>
+      </dependency>
+      <method_context>
+        <method_credential user="root" />
```

Comment:
Is this service available to the outside world as well? Any way to apply RBAC privileges at least? Could it run under hacluster user?